### ISSUE TITLE
MockComponents typescript signature

### ISSUE THREAD

**Issue by kekel87** [ike18t/ng-mocks#18]
Hello,
I have an issue when i trie to use `MockComponents` function.

```typescript
MockComponents(MyComponent, MySecondComponent, MyThirdComponent),
```
I get :
```
TS2345: Argument of type 'typeof MySecondComponent' is not assignable to parameter of type 'Type<MyComponent>'.
Type 'MySecondComponent' is not assignable to type 'MyComponent'.
    Property 'xxxx' is missing in type 'MySecondComponent'.
```

I think the signature is not correct but I can not find anything to propose.

To workaround, i write `MockComponents<any>(MapComponent, ViewComponent, LoaderComponent)` but it's dirty.

Thank you for your help.

**Comment by ike18t** [comment]
I was not able to reproduce this issue and I'm even using MockComponents in my own spec: https://github.com/ike18t/ng-mocks/blob/master/lib/mock-component/mock-component.spec.ts#L46

What version of @angular & typescript are you using?

**Comment by kekel87** [comment]
Yeah, I do not reproduce with your project.

My project versions :
```
Angular: 6.1.6
typescript: 2.9.2
```

The only difference is that your components are not in a NgModule (according to WebStrom).

**Comment by ike18t** [comment]
It appears to have been an issue where the compiler would only check against the first item in the array. I was able to reproduce the issue and had to make the type any. 😞 

New version has been released 69ebb9c36e85c77ccfe2cb7816f0f58ddd7d9cbf.  Let me know if you still run into the issue.
